networking_oneview/tests/unit/ml2/drivers/oneview/test_synchronization.py

```diff
@@ -90,7 +90,14 @@ def test_check_uplinkset_types_constraint_fails(self):
 
     @mock.patch.object(sync, 'get_session')
     def test_synchronize(self, mock_session):
+        functions_to_sync = 2
+        self.sync.synchronize()
+        self.assertEqual(functions_to_sync, mock_session.call_count)
```

Comment:
I know this is a way of doing it, but I think mocking the five methods and verifying that two of them were called and three were not lets it more clear.

Reply:
Agreed... can you rewrite this, @GustavoDinizMonteiro?